File: c++/1.1 Herencia Simple y Modificadores de acceso protegidos/1.1.2 Modificadores/PersonaV2Test.cpp
#include <iostream>

using namespace std;

class PersonaV2 {
  public: 
    string nombre;
    int edad;
    float estatura;

    string toString() {
      return "{nombre='" + nombre + "'" + 
            ", edad='" + std::to_string(edad) + "'" + 
            ", estatura='" + std::to_string(estatura) + "'" + 
            "}";
    }
};

int main(int argc, char const *argv[])
{
  PersonaV2 persona = PersonaV2();
  persona.nombre = "";
  persona.edad = -4;
  persona.estatura = 313.0f;

  cout << persona.toString() << endl;
  // {nombre='', edad='-4', estatura='313.0'}
  
  return 0;
}

File: python/1.1 Herencia Simple y Modificadores de acceso protegidos/1.1.2 Modificadores/PersonaV2Test.py
class PersonaV2:
  nombre = ""
  edad = 0
  estatura = 0.0
  
  def __str__(self):
    return f"[nombre='{self.nombre}', edad='{self.edad}', estatura='{self.estatura}']"

# Uso de la clase Persona
if __name__ == "__main__":
  persona = PersonaV2();
  persona.nombre = "";
  persona.edad = -4;
  persona.estatura = 313.0;
  
  print(persona);
  # [nombre='', edad='-4', estatura='313.0']